### DISCUSSION TITLE
AnimatedSprite2D: Set persistent grid values

### DISCUSSION THREAD

**Discussion by scotmcp** [godotengine/godot-proposals#8202] [2D]
When setting up sprint animations it is very often that the tilemap will be the same for multiple animations. The values here should persist, at least when opening the same file over and over again to create another animation. If you have to change it every time anyway, then does it matter if it persists when the next tilemap get's loaded? This would improve the workflow and quality of life by magnitudes.

![Untitled](https://github.com/godotengine/godot-proposals/assets/79280460/83cb9ee7-1f72-45c6-9aec-43ef25a1518b)

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/2588.
